cfg/agentinfo/info_test.go

```diff
@@ -80,42 +81,79 @@ func TestPlugins(t *testing.T) {
 	OutputPlugins = []string{"x", "y", "z"}
 
 	isRunningAsRoot = func() bool { return true }
-	plugins := Plugins()
+	plugins := Plugins("")
 	expected := fmt.Sprintf("inputs:(a b c) outputs:(x y z)")
 	if plugins != expected {
 		t.Errorf("wrong plugins string constructed '%v', expecting '%v'", plugins, expected)
 	}
 
 	isRunningAsRoot = func() bool { return false }
-	plugins = Plugins()
+	plugins = Plugins("")
 	expected = fmt.Sprintf("inputs:(a b c run_as_user) outputs:(x y z)")
 	if plugins != expected {
 		t.Errorf("wrong plugins string constructed '%v', expecting '%v'", plugins, expected)
 	}
+
+	plugins = Plugins("/aws/ecs/containerinsights/ecs-cluster-name/performance")
```

Comment:
Personally, I don't like that this test has multiple things checked in it, and this includes the `run_as_user` as a byproduct of the above scenario. Can we add an isolated test for just the change introduced?

Reply:
I see what you are referring to. Tbh, I was debating if I need to add this new test here as the `container_insights` flag is also tested/verified in the `TestUserAgent` below. The tests below just make this one a bit redundant. But if we still want to add test cases to `Plugins()`, do you think it's ok if we move it to between the 1st and 2nd test case in this function (so that the expected string will not contain `run_as_user`)